viewing_party/party.py

```diff
@@ -1,23 +1,121 @@
 # ------------- WAVE 1 --------------------
 
+from shutil import move

```

Comment:
Why did this line get included?

Reply:
I am actually not sure how this line got into my code! I had never heard of the shutil module until noticing this comment, so this is as much a mystery to me as it is to you!